uwg_schema/model.py

```diff
@@ -1,15 +1,421 @@
 """UWG Model schema."""
-from pydantic import Field, constr
+from pydantic import Field, validator, constr, conlist
+from typing import List
 
 from ._base import NoExtraBaseModel
+from .refDOE import BEMDef, SchDef, WEEK_MATRIX
 
 
-class Model(NoExtraBaseModel):
+class UWG(NoExtraBaseModel):
```

Comment:
@chriswmackey 

So just to confirm, do you mean create a class like (BoundaryLayerParam) that just contains the boundary layer variables - similar to DF?

I could do that. What we wouldn't be able to do is create the objects used by the UWG (i.e SimParam, Weather, geoParam)  ahead of time and pass them in, since they use the parameters don't discretely break down into separate classes.

Reply:
Yes, that 's what I was recommending but I understand that you want to keep the input schema close to what the UWG objects natively consume. The flat schema is fine and, if I want to make objects for `BoundaryLayerParameters` or `PavementParameters`, I can do them on the level of the `dragonfly-uwg` extension.

Reply:
Thanks. I do think we could improve the docstrings for these parameters at some future date, which would be one way of making the parameters more readable/useable. I noticed Bruno's 2012 thesis has some nicer descriptions then what ends up in the code.